apps/ospTutorial/CMakeLists.txt

```diff
@@ -42,7 +42,10 @@ target_include_directories(ospTutorialCpp SYSTEM
                                    ${LibJpegTurbo_INCLUDE_DIRS}
                                    ${CMAKE_CURRENT_BINARY_DIR})
 
-message(STATUS ${Boost_LIBRARIES})
+target_sources(ospTutorialCpp PRIVATE
+    connection.h
+    webscene.h
+)
```

Comment:
macht, dass die files in der IDE auftauchen, weil sie jetzt teil vom projekt sind